Makefile

```diff
@@ -113,12 +118,26 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	make -C conmon clean
 	make -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+local-cross:
+	@$(MAKE) $(CROSS_BUILD_TARGETS); \
+	for file in $(CROSS_BUILD_TARGETS); do test -s $$file && exit 1 ; done
+
+.PHONY: bin/crio.cross.%
+bin/crio.cross.%: .gopathok $(shell hack/find-godeps.sh $(GOPKGDIR) cmd/crio $(PROJECT))
```

Comment:
I think we want to drop the `find-godeps.sh` bit here (it's unnecessary for `.PHONY` targets).  I don't even think we want it for `bin/crio` (which as of the current master is not `.PHONY`), and I've filed #1590 to remove it there.

Reply:
yeah. I debated about this so that builds would depend on all the sources for timestamp freshness, but it did not accomplish that. I'll remove.